pkg/cmd/pipeline/export.go

```diff
@@ -0,0 +1,83 @@
+package pipeline
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+	"github.com/tektoncd/cli/pkg/cli"
+	"github.com/tektoncd/cli/pkg/export"
+	"github.com/tektoncd/cli/pkg/options"
+	"github.com/tektoncd/cli/pkg/pipeline"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	cliopts "k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+type ExportOptions struct {
+	Params       cli.Params
+	PipelineName string
+}
```

Comment:
I see that this struct is not used anywhere. Can you please remove it?

Reply:
the plan is to move this to common namespace and use this for Task/ClusterTask export when I or someone else get a bit more time to implement it,